bin/dockerfilelint

```diff
@@ -17,6 +17,9 @@ var argv = require('yargs')
     desc: 'Output linting results as JSON, equivalent to `-o json`.',
     type: 'boolean'
   })
+  .alias('v', 'version')
+  .version(function() { return require('../package').version; })
+  .describe('v', 'show version information')
```

Comment:
Looks like we'd need to use `'version'` as the key here for this to take effect. However, I'm ok using the default yargs description, so let's just drop this call to `.describe()`.